masterlist.yaml

```diff
@@ -1754,7 +1755,6 @@ plugins:
       - 'PerkusMaximus_Warrior.esp'
       - 'PerkusMaximus_CombatPlus.esp'
     tag:
-      - C.Regions
```

Comment:
Was removing this intentional?

Reply:
Yes its a bad tag caused by forwarded changes from unofficial patch. I forgot to add a note about it.

Reply:
OK